src/wasm/wasm-binary.cpp

```diff
@@ -106,6 +107,9 @@ void WasmBinaryWriter::finishSection(int32_t start) {
     assert(sizeFieldSize < MaxLEB32Bytes);
     std::move(&o[start] + MaxLEB32Bytes, &o[start] + MaxLEB32Bytes + size, &o[start] + sizeFieldSize);
     o.resize(o.size() - (MaxLEB32Bytes - sizeFieldSize));
```

Comment:
`MaxLEB32Bytes - sizeFieldSize` is now used in multiple places (also elsewhere), how about saving it in a local called `adjustment` or `diff` or such?